main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunnerConfig.java

```diff
@@ -88,6 +94,8 @@ public abstract static class Builder {
 
     public abstract Builder setCountryCode(CountryCode countryCode);
 
+    public abstract Builder setZonedDateTime(ZonedDateTime zonedDateTime);
```

Comment:
The method name is not very instructive. We know the type of what we are setting.
Maybe replace with something like `setDateForValidation`?